src/monitors/MonitorOptions.v3

```diff
@@ -44,21 +44,59 @@ component MonitorOptions {
 			var engine = Engine.new();
 			engine.extensions = Extension.set.all;
 			var tiering_override = if(SpcTuning.compileWhammModules, ExecuteOptions.getModeOrDefault("jit"), Execute.tiering);
-			var result = engine.loadWasmFileWithTiering(name, tiering_override); // TODO: limits
-			match (result) {
-				Ok(m) => {
-					monitors.put(WhammMonitor.new(m));
-					return;
+
+			// Split by '+'
+			var s = 0, pos = s;
+			var whammModuleName = "";
+			var otherModuleNames = Vector<string>.new();
+			while (pos < name.length) {
```

Comment:
I think this is a good candidate for adding to the Virgil `Strings` utility: `Strings.split(string, byte) -> Array<string>`. Wdyt?